src/main/scala/io/epiphanous/flinkrunner/FlinkRunner.scala

```diff
@@ -42,8 +44,7 @@ abstract class FlinkRunner[ADT <: FlinkEvent: TypeInformation](
     val executeJob: Boolean = true)
     extends LazyLogging {
 
-  val env: StreamExecutionEnvironment =
-    config.getStreamExecutionEnvironment
+  val env: StreamExecutionEnvironment = StreamExecutionEnvironment.getExecutionEnvironment
```

Comment:
So this was a change in 1.18?

Reply:
yes they changed the API,and they are deprecating StreamExecutionEnv after 1.18